### DISCUSSION TITLE
Add support for more than one eufy station

### DISCUSSION THREAD

**Discussion by Bassi44** [samemory/homebridge-eufy-security-control#392] [Ideas]
Hi,
Would be great to see that this plug-in is going to support more than one eufy stations.
In my scenario I have two stations to cover best connections for my cameras.
But with the plug-in I am only able to show one station in HomeKit.

cheers

Marc

**Comment by schliemann** [comment]
Good input. However this repo is closed. Future development will be done on the new repo.

Here is the new URL: https://github.com/homebridge-eufy-security/plugin

**Reply by Bassi44** [reply]
Thanks for the information. I will create the idea under the new url.
